docs/ehrql-new-tutorial-1a.md

```diff
@@ -59,110 +39,35 @@ Here, we import two components of Data Builder:
 In subsequent steps,
 we specify the data from the available data tables
 that we wish to add to the dataset.
 
-!!! warning
+### Find year of birth
+Next we define a year of birth. `date_of_birth` is in the patient table and therefore we can assign it to this new variable. We want to only capture the `year` of birth so we add the `.year` to the end of this variable assignment. 
 
-    The dataset used as the basis of an ehrQL query
-    is the dataset that is last to be given the name `dataset`.
+### Set population
+Finally we set the population of the dataset. We use the special method called `set_population()` and pass in the definition of the population. In this case, we want to use our previously created `year_of_birth` and say, if year of birth is equal to or greater than 2000, include in this dataset. 
 
-    A simple way to make your dataset definition clear is to
-    only use the name `dataset` for the query's dataset.
-
-!!! warning
-
-    Likewise, if you accidentally include multiple calls to `set_population()`,
-    it is the call that appears closest to the end of the dataset definition
-    that takes effect.
-
-    In future, calling `set_population()` more than once will cause a dataset definition to fail;
-    see the associated [Data Builder](https://github.com/opensafely-core/databuilder/issues/775) issue.
-
-!!! todo
-
-    Would it be simpler here to say:
-    "you probably should only have one dataset".
-    Are there cases where you might need multiple datasets within a definition?
-
-!!! todo
-
-    Consider using code annotations to describe the code.
-
-#### What happens when Data Builder generates a dataset?
-
-When the dataset definition is used by Data Builder to generate a dataset:
-
-1. The dataset definition is validated
-   to ensure that the resulting database query would be valid.
-2. A database query suitable for the specified database is created.
-3. The query is submitted to the database.
-4. Provided the query is successful, the query creates an output.
-
-In writing a *dataset definition* then,
-what we are really writing is a *database query*.
-Data Builder transforms the dataset definition into the appropriate database query,
-for the specific database.
-
-!!! note
-
-    You can see the database query that a dataset definition will generate
-    via the `databuilder --dump-dataset-sql` command.
-
-    For the minimal example `1a_minimal_dataset_definition.py` above,
-    the underlying SQL query generated is:
-
-    ```sql
-    SELECT patients.patient_id AS patient_id
-    FROM patients
-    WHERE CAST(STRFTIME('%Y', patients.date_of_birth) AS INTEGER) >= 2000;
-    ```
+## Your turn
+Run the dataset definition by:
 
-    !!! todo
+```
+opensafely exec databuilder:v0 generate-dataset "1a_minimal_dataset_definition.py" --dummy-tables "example-data/minimal/" --output "outputs.csv"
+```
 
-        This SQL should be kept updated.
+or if you are using `project.yaml`:
```

Comment:
thought: I'm not sure about retaining the alternative approach at all here; it's a legacy of `opensafely exec` not existing.

:+1: It's good because it's actually how someone would run a project.
:-1: It's bad because it's an alternative, and providing choices at this stage may not be helpful.

Reply:
How about we remove this and add a section in the tutorial - maybe section 8?